src/content/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model.mdx

```diff
@@ -92,19 +92,17 @@ To create a new access grant that gives a user group access to a role and an acc
-
 * **Account**: Choose this to be able to select from [the roles that are account-scoped](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles). These are the roles that have to do with using and configuring our platform features (and not about organization and user management).
 * **Organization**: Choose this to be able to select from [the roles that are organization-scoped](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles). These are the roles that govern organization- and user management. (Note that these users must also already belong to an account-scoped role. This is true for most users but if it's not, you may see a message that the user doesn't belong to an organization.)
 
 4. Select the **Role** you want to assign. Roles are organization-wide, so regardless of the authentication domain you're in, you have access to our [standard roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles) and any custom roles you've created. For tips on selecting roles, see the tips after these instructions.
-5. Select the **Account** you want to add access to from the dropdown. If you don't see an account that you'd expect to see, this may be for a few reasons. One is that you yourself don't have the proper permissions for that account. Another is that that account is not actually in your organization. For more information, see [Factors affecting access](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data/). If you are still having problems, talk to your account representative.
-6. If you want to continue adding more grants for that same group, select **Add another** at the bottom before clicking **Add access**.
-7. When you're done, if your users are already in the group you've added the grant to, they should have access within a few minutes (although for [EU region New Relic accounts](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), this can take up to twenty minutes or so). If your users are not yet in that group (which would be true if you just created an access grant with a new group), you'll need to go to the [**User management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks#where) and add one or more users to that group.
+5. Select the **Account** you want to add access to from the dropdown. If you don't see an account that you'd expect to see, this may be for a few reasons. One is that you yourself don't have the proper permissions for that account. Another is that that account is not actually in your organization. For more information, see [Factors affecting access](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data). If you are still having problems, talk to your account representative. 
```

Comment:
"to from" sounds a little odd. Maybe reword to "From the dropdown, select the Account where you want to add access." ?
Also consider changing "you yourself" to just "you"